tests/unit/setup-unit-test.ts

```diff
@@ -126,6 +129,17 @@ module.exports = async () => {
     }
   }
 
+  const cppFilePath = path.join(vcastEnvPath, cppFilename);
+  const createCppFile = `echo -- Environment: TEST > ${cppFilePath}`;
```

Comment:
The C++ files don't use this syntax

Reply:
Deleted in 9038de54b41c65eec73f8f620cbae868eb5a3936 .